### ISSUE TITLE
Multi-binary output?

### ISSUE THREAD

**Issue by hnghiem-nlp** [p-karisani/self_pretraining#1]
I wonder if this framework also works with multi-label classification instead of just binary? Can I just change the integer values in the train files for output accordingly?

Thank you

**Comment by p-karisani** [comment]
You probably already know this , but the code does not use the regular neural network LOOP-TRAIN-STEP pattern. It uses a framework that hosts BERT—it is implemented by myself.

The framework is technically capable of handling multi-class tasks, but the code on the Github repo doesn’t use that capability. Because the datasets in the paper are binary.

I will update the code for people like you who need a multi-class revision. But these days I am very busy, it will take time
